x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/types.ts

```diff
@@ -44,6 +57,7 @@ export interface RuleAlertParams {
   severity: string;
   tags: string[];
   to: string;
+  threats: ThreatParams[] | undefined | null;
```

Comment:
per comments below and how `ECS` labels it as `threat` I would keep it at `threat` even though yes it is indeed multiple threats.